spring-test/src/test/java/org/springframework/test/web/servlet/request/MockHttpServletRequestBuilderTests.java

```diff
@@ -31,7 +31,6 @@
 import jakarta.servlet.ServletContext;
 import jakarta.servlet.http.Cookie;
 import org.junit.jupiter.api.Test;
-
```

Comment:
Please reinstate the blank line.

Reply:
Oops, thank you for letting me know. I've fixed it. [6783610](https://github.com/spring-projects/spring-framework/pull/30497/commits/678361086da5e32bfc4de567d8353afe68bfb6a3)